src/lib/transaction_inclusion_status/transaction_inclusion_status.ml

```diff
@@ -38,9 +38,7 @@ let get_status ~frontier_broadcast_pipe ~transaction_pool cmd =
         |> Mina_block.Validated.valid_commands
         |> List.exists ~f:(fun { data = found; _ } ->
                let found' = User_command.forget_check found in
-               User_command.equal_ignoring_proofs_and_hashes cmd found'
```

Comment:
This change is perhaps ok. Originally I tried to retain the previous behavior as much as possible. But now I see that in case of transaction being in the pool, we'll only check transaction hash, and that doesn't rely on contents of proofs.

The only thing that might be different is that someone will submit their transactions into a block, and then recreate it (same contents, new proofs) and it will be said as included, although nobody has seen exactly this sequence of bytes before.

Sounds like something someone could potentially misuse, but nothing convincing comes to my mind as a demonstration of this potential.